docs/howto/settings.rst

```diff
@@ -165,10 +155,6 @@ Search
 
 .. code-block:: python
 
-  # Override the search results template for wagtailsearch
-  WAGTAILSEARCH_RESULTS_TEMPLATE = 'myapp/search_results.html'
-  WAGTAILSEARCH_RESULTS_TEMPLATE_AJAX = 'myapp/includes/search_listing.html'
```

Comment:
Need to update this here too: http://docs.wagtail.io/en/latest/core_components/search/searching.html